crates/rpc/src/lib.rs

```diff
@@ -18,6 +21,27 @@ impl BeerusRpc {
     }
 
     pub async fn run(self) -> Result<(SocketAddr, ServerHandle), Error> {
+        let local_spec_version =
+            self.spec_version().await.map_err(BeerusRpcError::from).unwrap();
```

Comment:
I'd be interested in getting rid of these `unwrap`s.
I suppose the one for the local version would just never happen, if that's the case then an `expect` saying it should never happen would be nicer.